plugin.video.southpark_unofficial/default.py

```diff
@@ -309,7 +315,7 @@ def getVideoData(mediagen):
 	if audio == "de":
 		mediagen += "&deviceOsVersion=4.4.4&acceptMethods=hls";
 		mediagen = mediagen.replace('{device}', 'Android')
-	print mediagen
+	print "MEDIAGEN: " + mediagen
```

Comment:
please convert all these print statements to xbmc.log for next update.
https://codedocs.xyz/xbmc/xbmc/group__python__xbmc.html#ga3cd3eff2195727144ad6fbabd4744961